README.md

```diff
@@ -2,6 +2,15 @@
 
 A golang client library for [AWX](https://github.com/ansible/awx) and [Ansible Tower](https://www.ansible.com/products/tower) REST API.
 
+## Build
+
+In order to build this package `glog` package is needed. run:
```

Comment:
Sorry for being picky, but he next workd after a dot should start with uppercase: ... needed. Run ...

Reply:
Fixed